utils/src/lib.rs

```diff
@@ -404,10 +405,36 @@ async fn wait_for_transaction_to_be_mined<P: JsonRpcClient>(
         sleep(interval).await;
     }
 
-    tracing::error!("contraact call {hash:?}: not mined after {retries} retries");
+    tracing::error!("contract call {hash:?}: not mined after {retries} retries");
     false
 }
 
+pub async fn wait_for_anvil_endpoints<P: JsonRpcClient>(provider: &Provider<P>) {
+    // When we are running a local Anvil node, as in tests, some endpoints (e.g. eth_feeHistory)
+    // do not work until at least one block has been mined. Send a transaction to force the
+    // mining of a block.
+    provider
+        .send_transaction(
+            TransactionRequest {
+                to: Some(Address::zero().into()),
+                ..Default::default()
+            },
+            None,
+        )
+        .await
+        .unwrap()
+        .await
+        .unwrap();
+
+    // When we are running a local Anvil node, as in tests, some endpoints (e.g. eth_feeHistory)
+    // do not work until at least one block has been mined. Send a transaction to force the
+    // mining of a block.
```

Comment:
This comment is a duplicate of the above?

Reply:
Right, removed in ba28632ad033e86795ff111859009832bf12b38a.